studies/forms.py

```diff
@@ -279,3 +279,18 @@ def __init__(self, *args, **kwargs):
         """
         super(SessionStartForm, self).__init__(*args, **kwargs)
         self.fields['sessions_number'].required = True
+    
+    def clean(self):
```

Comment:
Wouldn't it be better to use a ``MaxValueValidator`` (either  in the form or on the actual model field itself)

Reply:
Yes, it would. I've moved this requirement to the model now with a validator.